examples/records_with_no_created_at_timestamp/connector.py

```diff
@@ -0,0 +1,91 @@
+# This is an example to show how to work with records where the source does not provide a created_at(or equivalent) field.
```

Comment:
do we need to keep this example here, or move it to one of the 3 folders?

Reply:
Moved to `Common patterns for connector` folder